### PR TITLE
Playground UX fixes

### LINKED ISSUE #105: chat-history in playground
starts with an empty human message - imo should be just completely empty
Screenshot 2023-10-25 at 1 08 35 PM

### PR DESCRIPTION
**Pull request by dqbd**
- [x] Reset button
 - [x] Remove default single message
 - [x] Remove default `anyOf` handling
 - [x] `additional_kwargs` are not shown in the output
 - [x] Cmd+Enter on submit
 - [x] Loading spinner

Closes #105, #110

### CODE REVIEW

File: langserve/playground/src/components/StreamOutput.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+// inlined from langchain/schema
+interface BaseMessageFields {
+  content: string;
+  name?: string;
+  additional_kwargs?: {
+    [key: string]: unknown;
+  };
+}
+
+class AIMessageChunk {
```
Comment (eyurtsev) [inline langserve/playground/src/components/StreamOutput.tsx:12]: Do we need to handle FunctionMessageChunk later?

File: langserve/playground/src/components/CustomAnyOfRenderer.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  const anyOfRenderInfos = createCombinatorRenderInfos(
+    (props.schema as JsonSchema).anyOf!,
+    props.rootSchema,
+    "anyOf",
+    props.uischema,
+    props.path,
+    props.uischemas
+  );
+
+  const selectedIndex = anyOfRenderInfos.length - 1;
```
Comment (eyurtsev) [inline langserve/playground/src/components/CustomAnyOfRenderer.tsx:20]: comment may not make sense -- i have no idea what the code is supposed to do

Why is this hard-coded to the last index?

File: langserve/playground/src/App.tsx [hunk 3]
```diff
@@ -341,46 +202,101 @@ function App() {
...
             <div className="p-4 border border-divider-700 flex flex-col gap-3 rounded-2xl bg-background">
-              <h3 className="font-medium">Inputs</h3>
+              <div className="flex items-center justify-between">
+                <h3 className="font-medium">Inputs</h3>
+                {isInputResetable && (
+                  <button
+                    type="button"
+                    className="text-sm px-1 -mr-1 py-0.5 rounded-md hover:bg-divider-500/50 active:bg-divider-500 text-ls-gray-100"
+                    onClick={() =>
+                      setInputData({
```
Comment (eyurtsev) [inline langserve/playground/src/App.tsx:290]: :heart:

File: langserve/playground/src/App.tsx [hunk 4]
```diff
@@ -464,18 +380,35 @@ function App() {
...
+                    <div role="status">
+                      <svg
+                        aria-hidden="true"
+                        className="w-5 h-5 animate-spin text-white fill-ls-blue"
+                        viewBox="0 0 100 101"
+                        fill="none"
+                        xmlns="http://www.w3.org/2000/svg"
+                      >
+                        <path
+                          d="M100 50.5908C100 78.2051 77.6142 100.591 50 100.591C22.3858 100.591 0 78.2051 0 50.5908C0 22.9766 22.3858 0.59082 50 0.59082C77.6142 0.59082 100 22.9766 100 50.5908ZM9.08144 50.5908C9.08144 73.1895 27.4013 91.5094 50 91.5094C72.5987 91.5094 90.9186 73.1895 90.9186 50.5908C90.9186 27.9921 72.5987 9.67226 50 9.67226C27.4013 9.67226 9.08144 27.9921 9.08144 50.5908Z"
```
Comment (eyurtsev) [inline langserve/playground/src/App.tsx:400]: does this draw a parrot?

File: langserve/playground/src/components/CustomAnyOfRenderer.tsx (hunk 2 above)
Comment (dqbd) [reply langserve/playground/src/components/CustomAnyOfRenderer.tsx:20]: The idea here is to get rid of the tabs altogether, as it doesn't make sense to show all possible union options in a nice way and if I recall correctly, only non-prompt inputs do include `anyOf` in our templates. 

Can revisit later with a dropdown style input instead, but this should work fine for now.

File: langserve/playground/src/components/StreamOutput.tsx (hunk 1 above)
Comment (dqbd) [reply langserve/playground/src/components/StreamOutput.tsx:12]: Good catch, need to handle FunctionMessageChunk as well

**Pull request merged**